samples/DashboardDemo/src/DashboardDemo.Application.Contracts/CountersWidgetInputDto.cs

```diff
@@ -2,10 +2,7 @@
 
 namespace DashboardDemo
 {
-    public class CountersWidgetInputDto
+    public class CountersWidgetInputDto : WidgetFilter
```

Comment:
Remove WidgetFilter and duplicate StartDate  and EndDate, it is not a problem to duplicate here.

Reply:
Do it in the dev branch, I've merged it.